book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I suggest that you make `rentals` as an `attr_accessor` because we need to read and modify it